specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-03-01-preview/definitions.json

```diff
@@ -5035,6 +5035,63 @@
       ],
       "description": "Sign-In settings for the Developer Portal."
     },
+    "PolicyRestrictionCollection": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PolicyRestrictionContract"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Next page link if any."
+        }
+      },
+      "description": "The response of the get policy restrictions operation."
+    },
+    "PolicyRestrictionUpdateContract": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyRestrictionContractProperties",
+          "description": "Properties of the Policy Restriction."
+        }
+      },
+      "description": "Policy restriction contract details."
+    },
+    "PolicyRestrictionContract": {
+      "type": "object",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/PolicyRestrictionContractProperties",
+          "description": "Properties of the Policy Restriction."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./../../../../../common-types/resource-management/v2/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "Policy restriction contract details."
+    },
+    "PolicyRestrictionContractProperties": {
+      "type": "object",
+      "properties": {
+        "scope": {
+          "type": "string",
+          "description": "Path to the policy document."
+        },
+        "requireBase": {
+          "type": "boolean",
+          "description": "Indicates if base policy should be enforced for the policy document."
+        }
```

Comment:
Consider making this an ENum as is called out in the Swagger LintDiff errors

Reply:
Would this mean I have to update the code as well?

Reply:
updated